bccsp/sw/fileks.go

```diff
@@ -355,27 +359,51 @@ func (ks *fileBasedKeyStore) loadKey(alias string) ([]byte, error) {
 func (ks *fileBasedKeyStore) createKeyStoreIfNotExists() error {
 	// Check keystore directory
 	ksPath := ks.path
-	missing, err := utils.DirMissingOrEmpty(ksPath)
 
-	if missing {
-		logger.Debugf("KeyStore path [%s] missing [%t]: [%s]", ksPath, missing, utils.ErrToString(err))
+	// Check if keystore directory exists
+	_, err := os.Stat(ksPath)
+	if os.IsNotExist(err) {
```

Comment:
```suggestion
	if err != nil && !os.IsNotExist(err) {
	    return err
	}
```
It helps to reduce the extra indentation.